HelpSource/Classes/Image.schelp

```diff
@@ -692,3 +699,19 @@ For example, in a high DPI context, the pixel ratio of a View might be 2. When d
 By default, the pixelRatio of all Images is 1 - emphasis::this ensures that an image will look the same when drawn on a normal or a high DPI view::. Setting a custom (!= 1) pixelRatio should generally only be done to draw specially rendered high DPI images to a View that is known to be high DPI.
 
 Note that when drawing to an Image using link::Classes/Pen::, pixelRatio is accounted for - so, a line of width 1 will have a true width of 1px for an image where code::image.pixelRatio==1::, and a true width of 2px where code::image.pixelRatio==2::.
+
+
+PRIVATE::copy
+PRIVATE::fill
+PRIVATE::isValid
+PRIVATE::name
+PRIVATE::prNewEmpty
+PRIVATE::prNewFromWindow
+PRIVATE::prNewPath
+PRIVATE::prNewSVG
+PRIVATE::prNewURL
+PRIVATE::prSetPainter
+PRIVATE::prSetSize
+PRIVATE::prTransferPixels
+PRIVATE::prUnsetPainter
+PRIVATE::smooth
```

Comment:
please add a newline to the end of the file. also, should all of these really be hidden? `copy`, `fill`, etc. seem like important methods

Reply:
Good point, I'll prevent non `pr*` methods from being hidden.